build.sbt

```diff
@@ -4,28 +4,30 @@ lazy val scala212               = "2.12.10"
 lazy val scala213               = "2.13.5"
 lazy val supportedScalaVersions = List(scala212, scala213)
 
+ThisBuild / credentials += Credentials(
+  "emnify/emnify-maven",
+  "emnify-648956897802.d.codeartifact.eu-west-1.amazonaws.com",
+  "aws",
+  sys.env.getOrElse("CODEARTIFACT_AUTH_TOKEN", "Unknown")
+)
+
 ThisBuild / scalaVersion := "2.13.5"
 ThisBuild / organization := "io.github.kirill5k"
 ThisBuild / organizationName := "example"
 ThisBuild / resolvers ++= Seq(
-  "Confluent" at "https://packages.confluent.io/maven/",
   Resolver.mavenLocal,
-  Resolver.sonatypeRepo("public"),
-  Resolver.sbtPluginRepo("releases")
+  Resolver.sbtPluginRepo("releases"),
+  "Confluent" at "https://packages.confluent.io/maven/",
+  "emnify/emnify-maven" at "https://emnify-648956897802.d.codeartifact.eu-west-1.amazonaws.com/maven/emnify-maven/",
```

Comment:
what happens if you're not logged in to AWS?

Reply:
I don't know sbt so well but from what I understood and observed, when you're not logged in you'll get unauthorized if it tries to fetch from this repo. But maven public fallback is not deactivated here, so it won't try to always go through this one, only if the others don't give the artifacts.

But I guess if we want to use it as a mirror of maven public, I probably should disable the maven public fallback right?

The other thing that bothered me here, is the confluent repo, we'll still be dependent on this one because I can't have a codeartifact upstream that points to it. Codeartifact only supports a limited predefined list of public repositories that it can proxy to.